rllib/env/env_runner.py

```diff
@@ -3,6 +3,9 @@
 
 from ray.rllib.utils.actor_manager import FaultAwareApply
 from ray.rllib.utils.annotations import ExperimentalAPI
+from ray.rllib.utils.framework import try_import_tf
+
+tf1, _, _ = try_import_tf()
```

Comment:
Add a TODO that in the future, we'll make EnvRunner framework agnostic?
Actually, this should already be agnostic in this PR, correct?

We only need tf to translate numpy arrays to tensors for the RLModule inputs and should therefore outsource this step into a framework-agnostic utility function call.